### ISSUE TITLE
password reset mail empty

### ISSUE THREAD

**Issue by selyod-ka** [OpenSlides/openslides-backend#2695]
**Describe the bug**
When using the password reset link on firefox, a mail is sent to the correct mail-address, but the mail is empty.

**How to Reproduce**
1. Go to 'https://openslides-domain.tld/login/forget-password
2. Enter mail-address, press button.
3. Mail is sent to the address.
4. The mail is empty; the subject reads as ": <username>"

**Expected behavior**
The mail should contain a password-reset-link.

**System information**
- OpenSlides version: 4.1.14
- Browser: Firefox 
- Browser version: 129.0.2

**Additional context**
With Brave (=Chromium 127) the mail sent contains a password reset link.

iiuc, the logs do not contain any useful information and they are the same for both password reset attempts (with firefox and with brave).

```
docker logs -f
[...]
backendAction-1     | INFO:openslides_backend.shared.handlers.base_handler:Performing action user.forget_password.
auth-1              | 2024-09-03T11:28:01.601Z http://auth:9004: POST -- /internal/auth/create-authorization-token
datastoreWriter-1   | [2024-09-03 11:28:01 +0000] [22] [INFO] Created position 22
datastoreWriter-1   | [2024-09-03 11:28:01 +0000] [22] [INFO] Events executed (1 UPDATE)
datastoreWriter-1   | [2024-09-03 11:28:01 +0000] [22] [INFO] UPDATE: ['user/3']
backendAction-1     | INFO:openslides_backend.shared.handlers.base_handler:Request was successful. Send response now.
[...]
```

**Comment by selyod-ka** [comment]
sorry, somehow i did not receive any notification on this ticket's update.

i just tested again, with firefox 131 and https://demo.os4.openslides.com
the post-request to https://demo.os4.openslides.com/system/action/handle_request is:
`
[{"action":"user.forget_password","data":[{"email":"redacted@example.com"}]}]`

the mail i received now contains the username in the subject, but the mail body is still empty.

```
Return-Path: <noreply@openslides.com>
Delivered-To: user@example.com
Received: from proxy02.example.name ([127.0.0.1])
	by dovecot03.example.name (Dovecot) with LMTP id 6gnxMIxtG2dBxQIAVYRftg
	for <user@example.com>; Fri, 25 Oct 2024 12:11:05 +0200
Received: from proxy02.example.com ([127.0.0.1])
	by proxy02.example.name (Dovecot) with LMTP id Et2UM/I+G2e9iAEAGFAyLg
	; Fri, 25 Oct 2024 12:11:05 +0200
Received: from mailin04.example.com (unknown [10.0.0.64])
	by proxy02.example.com (Postfix) with ESMTPS id 4XZdpd5Jq5z125J
	for <user@example.com>; Fri, 25 Oct 2024 12:11:05 +0200 (CEST)
Received: from mx04.example.com (mailin04.example.com [127.0.0.1])
	by mailin04.example.com (Postfix) with ESMTPS id A7A6021E3D
	for <user@example.com>; Fri, 25 Oct 2024 12:11:05 +0200 (CEST)
X-Virus-Scanned: amavisd-new at example.com
 d=CQSKPMLleoBo3Fr3aLJepSGemenrl+FYFCC//BczgauhLule8qfLc09FZYx7MtA5Z8ONzvT9W
 pUQ7KCwHdGUleZqME1axCmM2gHKqiNrmjtpNP33ZK7jGtpmnUNMJg7YCibtqW3pLqXsJmJpzqI3
 h6nk+acmnc39q50LzD8FA/HYUkPvCZzWuC3xyD6qy0QyZ9JFJiOWF4IXjWlFEc17Xvr/LUqhRYS
 Ud+ypQWQ2E10WPWOWIofjAeoQo04vvDthdOWUKxBAKJeF3HVv/PDi3Ga/pGv/rE4MDjBxIYTSSA
 IHNQ+XwW+nUVI4ahhzfYiE9ZVl8Mr1YzLhUGP8PS64p3BiipgEkUTipDqhJBScBKmoh7S7RT7M2
 5NtxMg/2mEBe6bIx3QXDQFahggiEocxoj4pxTQvr2NrOpvGXzuHrZbRWg3xX7zDWo6GMD835qhx
 TDHqo2VQ5AvQhlg/a3TVHDAqdlRULdFq7ziCiD94F3y/EilRnDZjn8w=
Authentication-Results: example.com; dmarc=pass (p=none dis=none) header.from=openslides.com
Authentication-Results: example.com; spf=pass smtp.mailfrom=openslides.com
Authentication-Results: example.com;
	dkim=pass (2048-bit key) header.d=openslides.com header.i=@openslides.com header.b=tVsdnhXT;
	dkim-atps=neutral
Received: from mail.openslides.com (mail.openslides.com [136.243.0.249])
	by mx04.example.com (Postfix) with ESMTPS id 4XZdpc4mSlz12Lc
	for <redacted@example.com>; Fri, 25 Oct 2024 12:11:04 +0200 (CEST)
Received: from [172.18.0.10] (openslides-cloud3.intevation.de [176.9.30.134])
	by mail.openslides.com (Postfix) with ESMTPS id 9BDF78044B
	for <redacted@example.com>; Fri, 25 Oct 2024 12:10:58 +0200 (CEST)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=openslides.com;
	s=201904; t=1729851058;
	bh=frcCV1k9oG9oKj3dpUqdJg1PxRT2RSN/XKdLCPjaYaY=;
	h=From:To:Subject:Date:From;
	b=tVsdnhXTb77Ao/uaAgQjw6S3w2fokdOx3abzb0Qr25yldGo/AHKykQXeebE6JG615
	 0vtjdDvTM5VvWbfNSshpQoLavlSEy4CKqgNOUZTCuzwlyvGOaKBH4YlfAVqKFjFIEt
	 LOyhtusfzVz/RcGnMcCH7LcSmH6yhalOE4X4k4LhmHV785PtQCD0FDS2JWAOXsZosr
	 O9YHGFj1aUf+xPjsRE+a54WaeH6A2ZXXVMBu3fFGCxWYm6jYDR6vbkCOUq2msGHFsM
	 XIVITt6MdA4JsVRwe6yadAOmkkRQQU6qz8oHLzgCDVUmKY7ITqYRhsSBgs/sDKctO5
	 uP4nLWXMz9xAA==
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 7bit
MIME-Version: 1.0
From: noreply@openslides.com
To: redacted@example.com
Subject: : testtest9734
Date: Fri, 25 Oct 2024 10:10:58 -0000
Message-ID: <172985105861.12.5160393747647820599@mail.openslides.com>
```

**Comment by MSoeb** [comment]
This issue seems to be solved by #2701. 

Note: If it still happens in your configuration, please create another issue. I strongly assume, that reason of this issue is the same like in #2697 and the solution in #2701 will also fix this problem.
